apps-rendering/src/components/editions/shareIcon/index.tsx

```diff
@@ -86,18 +85,22 @@ const ShareIcon: FC = () => {
 	}, []);
 
 	return showIcon ? (
-		<button
-			css={buttonStyles}
-			onClick={(): void =>
-				pingEditionsNative({ kind: MessageKind.Share })
-			}
-		>
-			{platform === Platform.IOS ? (
-				<IOSShareIcon />
-			) : (
-				<AndroidShareIcon />
-			)}
-		</button>
+		<div css={hydratedButtonStyles}>
+			<span className="js-share-button" role="button">
```

Comment:
Do you need both the `div` and the `span`? Can the styles, class and role all go onto the `div`?

Reply:
Good point, we can! Amended as per.